projects/ion/src/lib/input-counter/input-counter.component.scss

```diff
@@ -4,13 +4,30 @@
   display: inline-flex;
   border: 1px solid $neutral-5;
   border-radius: 6px;
+  overflow: hidden;
+
+  ::ng-deep .ion-btn-ghost {
+    border-radius: 0;
+
+    &:disabled {
+      background-color: $neutral-2 !important;
+      pointer-events: none;
+      cursor: not-allowed;
+    }
```

Comment:
```suggestion
    &:disabled, &:disabled:hover {
      background-color: $neutral-2 !important;
      cursor: not-allowed;
    }
      cursor: not-allowed;
    }
```
![Captura de tela 2024-10-04 07:59:44](https://github.com/user-attachments/assets/31bb7bdc-0b2c-41a3-adce-cfa222f08f27)
remove the pointer-events and add a validation in the functions. It is more reliable and this property removes the cursor change.